driver/CMakeLists.txt

```diff
@@ -26,14 +26,14 @@
 
 find_package(Threads REQUIRED)
 
-add_executable(MIOpenDriver main.cpp InputFlags.cpp)
+add_executable(MIOpenDriver main.cpp InputFlags.cpp rocrand_wrapper.cpp)
 if(WIN32)
     # Refer to https://en.cppreference.com/w/cpp/language/types for details.
     target_compile_options(MIOpenDriver PRIVATE $<BUILD_INTERFACE:$<$<CXX_COMPILER_ID:Clang>:-U__LP64__>>)
 endif()
 add_dependencies(MIOpenDriver generate_kernels)
 target_include_directories(MIOpenDriver PRIVATE ../src/kernels)
-target_link_libraries(MIOpenDriver MIOpen Threads::Threads)
+target_link_libraries(MIOpenDriver MIOpen Threads::Threads rocrand)
```

Comment:
@atamazov found an weird issue with
```
[100%] Linking CXX executable ../bin/MIOpenDriver
ld.lld: error: unable to find library -lrocrand
```
here is how to reproduce:
- rocm/miopen:ci_a9eebf
- CXX=/opt/rocm/llvm/bin/clang++ CXXFLAGS='-Werror'  cmake -DMIOPEN_TEST_FLAGS=' --disable-verification-cache ' -DCMAKE_BUILD_TYPE=debug -DCMAKE_CXX_FLAGS_DEBUG='-g -fno-omit-frame-pointer -fsanitize=undefined -fno-sanitize-recover=undefined -Wno-option-ignored ' -DBUILD_DEV=On -DMIOPEN_USE_MLIR=ON -DMIOPEN_GPU_SYNC=Off -DMIOPEN_BACKEND=HIPNOGPU -DMIOPEN_INSTALL_CXX_HEADERS=On -DCMAKE_PREFIX_PATH=/opt/rocm ..
- make -j$(nproc)

Reply:
@junliume Thanks, I'll work on it. FYI with ci_d868bf everything is fine.

Reply:
> @junliume Thanks, I'll work on it. FYI with ci_d868bf everything is fine.

@atamazov  then likely manually installing `rocrand` in the docker has messed up the library.

Reply:
@junliume The reason of the issue is `-DMIOPEN_BACKEND=HIPNOGPU`. Which seems logical (as rocrand uses GPU). I am going to disable building the driver for the HIPNOGPU backend (because what's the use for the driver without GPU?)

Reply:
@junliume Done in 671215b67